src/pyJiraCli/cmd_import.py

```diff
@@ -141,13 +152,13 @@ def _create_issues(jira: JIRA,
 
         if external_id is None:
             ret_status = Ret.CODE.RET_ERROR
-            printer.print_error(
+            LOG.print_error(
                 PrintType.ERROR, "External ID must be specified.")
```

Comment:
print error does not take string as an argument

Reply:
the main function should be responsible to print an error at the end, use ret_status to return the error